docs/packaging.rst

```diff
@@ -98,4 +98,4 @@ Examples
 
 * `arch linux <https://www.archlinux.org/packages/community/any/python-hypothesis/>`_
 * `fedora <https://admin.fedoraproject.org/pkgdb/package/rpms/python-hypothesis/>`_
```

Comment:
While you're here, could you update Fedora link? It should  now be: https://src.fedoraproject.org/rpms/python-hypothesis.